README.md

```diff
@@ -219,6 +220,7 @@ Content-Type: application/json
   "origin": "local",
   "type": "response",
   "correlation": "2d66e4bc-9a0d-11e5-a84c-1f39510f0d6b",
+  "duration": 25,

```

Comment:
ms implied.
I still feel taht we have to find better name for log message since `duration` could mislead users to think about total request/response duration. What about `correlation_duration`?